### ISSUE TITLE
pip error when trying to install version 1.9.1

### ISSUE THREAD

**Issue by ray-5050** [apache/incubator-mxnet#21178]
## Description
when I use pip install mxnet, it tries to install version 1.7.0 and crashes due to trying to install an old numpy version.
When I try to install the current version, I get the error indicated below.  Thanks for your help!

### Error Message
ERROR: Could not find a version that satisfies the requirement mxnet==1.9.1 (from versions: 0.11.0, 0.12.0, 0.12.1, 1.0.0, 1.0.0.post1, 1.0.0.post3, 1.0.0.post4, 1.1.0.post0, 1.2.0, 1.2.1, 1.2.1.post1, 1.3.0, 1.3.1, 1.4.0, 1.4.0.post0, 1.4.1, 1.5.0, 1.6.0, 1.7.0.post1, 1.7.0.post2)
ERROR: No matching distribution found for mxnet==1.9.1

## To Reproduce
(If you developed your own code, please provide a short script that reproduces the error. For existing examples, please provide link.)

### Steps to reproduce
(Paste the commands you ran that produced the error.)

1.  pip install mxnet
2. pip install mxnet==1.9.1

## What have you tried to solve it?

1. indicating that I want version 1.9.1
2. start with a new venv and still happens, cleared all pip caches, no change

## Environment
Windows 11, python 3.10.9, pip 22.3.1

<details>
<summary>Environment Information</summary>

```
# Paste the diagnose.py command output here
```

</details>

**Comment by dai-ichiro** [comment]
~~~
pip install numpy==1.23.5
pip install mxnet -f https://dist.mxnet.io/python/cpu
~~~

mxnet==1.8.0 will be installed.
On Windows, the latest version that can be installed with pip is mxnet==1.8.0.

**Comment by Prakhar-Gupta-03** [comment]
I'm sorry, this might not be relevant to the thread, but, for installing mxnet with NVIDIA GPU on windows, which is the latest version of mxnet available?

**Comment by Maddill** [comment]
have you check your cuda version? if you are using cpu, maybe mxnet 1.9.1 not yet compatible.
you can check your device compability here https://mxnet.apache.org/versions/1.9.1/get_started?platform=windows&language=python&processor=cpu&environ=pip&

**Comment by Prakhar-Gupta-03** [comment]
Yeah, I had used it later on and using mxnet 1.8.0 with cuda 10.2 since then, thanks!

**Comment by francisco-camargo** [comment]
> ```
> pip install numpy==1.23.5
> pip install mxnet -f https://dist.mxnet.io/python/cpu
> ```
> 
> mxnet==1.8.0 will be installed. On Windows, the latest version that can be installed with pip is mxnet==1.8.0.

Spent a day trying to get mxnet to install on windows, this was the only thing that worked for me!!
